public/react/SaveReports/components/SaveReportModal/SaveReportModal.test.tsx

```diff
@@ -65,6 +65,7 @@ describe("<SaveReportModal />", () => {
           validate={mockValidate}
           isCreating={false}
           setName={vi.fn()}
+          autoFocus={false}
```

Comment:
not sure why, but autoFocus was causing this test to fail because it couldn't find the invalid error text. Spent some time trying to figure out what the deal was but it wasn't making sense. Open to suggestions but otherwise just added a param to disable it

Reply:
Do you have the error message?

Reply:
```Unable to find an element with the text: Invalid name. This could be because the text is broken up by multiple elements. In this case, you can provide a function for your text matcher to make your matcher more flexible.```

can see it by setting autoFocus to true as well